test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildProcessManager.cs

```diff
@@ -93,20 +94,33 @@ internal static class MSBuildProcessManager
                 }
 
                 process.WaitForExit();
-                var result = new MSBuildResult(project, process.StartInfo.FileName, process.StartInfo.Arguments, process.ExitCode, output.ToString());
+
+                string outputString;
+                lock (outputLock)
```

Comment:
#yolo

Reply:
Were you able to verify if this fixes the flakiness somehow or is it just truly #yolo?